data/mods/MindOverMatter/monsters/monsters_spells.json

```
@@ -31,6 +31,24 @@
     "min_range": 10,
     "max_range": 10
   },
+  {
+    "id": "pigeon_aura",
+    "type": "SPELL",
+    "name": "Pigeon aura",
+    "description": "You consume human blood to keep the thirst at bay.",
```

```suggestion
    "description": "Anyone nearby will get stronger.",
```
Those are some hardcore pigeons

---

Oops thanks for catching that!